mmv1/third_party/terraform/services/kms/data_source_google_kms_key_rings.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
```suggestion
```

Reply:
what's the process for this? I see some data sources have it and some dont. we should clean this up in the future.

https://github.com/GoogleCloudPlatform/magic-modules/blob/f514cc2e5b57129f3136b17ce602aa724fa390c4/mmv1/third_party/terraform/services/storage/data_source_google_storage_buckets.go#L1-L5